signingscript/docker.d/passwords.yml

```diff
@@ -233,6 +233,13 @@ in:
              ["autograph_apk"],
              "autograph"
             ]
+        project:mobile:fenix:releng:signing:cert:fennec-production-signing:
```

Comment:
colons in bare strings make me nervous, but yaml knows better how to not parse them :D